### DISCUSSION TITLE
Add a convolution reverb audio effect.

### DISCUSSION THREAD

**Discussion by catboy-catfish** [godotengine/godot-proposals#9389] [General Discussions]
It would be cool if you guys could consider adding a convolution/impulse-response reverb. The algorithmic reverb in Godot is cool and all but it does get old sometimes

**Comment by mieldepoche** [comment]
see https://github.com/godotengine/godot-proposals/issues/1721

**Reply by catboy-catfish** [reply]
Thanks for referring me, I might close this now
